bin/fuel-core/src/cli.rs

```diff
@@ -87,6 +103,11 @@ pub async fn init_logging() -> anyhow::Result<()> {
 }
 
 pub async fn run_cli() -> anyhow::Result<()> {
+    init_logging().await?;
+    if let Some(path) = init_environment() {
+        let path = path.display();
```

Comment:
Is it possible to add a test to verify that it works properly with the `.env` file? Maybe even we can have a snapshot of `.env` file to track all future modificaitons?